modules/nlu/src/translations/en.json

```
@@ -74,5 +74,6 @@
     "tagSelectionMessage": "Click on a slot or use numbers as keyboard shortcuts"
   },
   "title": "Language Understanding",
-  "trainNow": "Train now"
+  "trainNow": "Train now",
+  "retrainAll": "Retrain all"
```

I'm sorry to change my mind again.  The user has no idea that the training happens in multiple steps or has multiple components that can be cached, **training is just one big thing that needs to be done so their chatbot can get smarter**.

Let's change this for Force train. And if users ever come with some questions about the inner workings then we'll add a tooltip or something.

```suggestion
  "retrainAll": "Force train"
```